Marlin/language_de.h

```diff
@@ -88,6 +93,7 @@
 #define MSG_PID_I                           "PID I"
 #define MSG_PID_D                           "PID D"
 #define MSG_PID_C                           "PID C"
+#define MSG_SELECT                          "Auswählen"
 #define MSG_ACC                             "A"
 #define MSG_VXY_JERK                        "V XY Jerk"
 #define MSG_VZ_JERK                         "V Z  Jerk"
```

Comment:
(i'll stick with english so the others can weigh in)
Point taken, I didn't know about this, however, I would further argue if the English Marlin community justifies assigning the word Jerk to this behavior in Marlin I believe we should follow suit in the German translation.